packages/api-kit/index.ts

```diff
@@ -0,0 +1,57 @@
+import express, { Express } from "express";
+
+import type {
+  AppConfigOptions,
+  BodyParserType,
+} from "./src/interfaces/app-config-options.interface";
+
+let app: Express;
+
+let config: AppConfigOptions = {
+  cors: !!process.env.API_CORS || false,
+  bodyParser: (process.env.API_BODY_PARSER as BodyParserType) || "json",
+};
+
+function createApp() {
+  app = express();
+
+  if (process.env.NODE_ENV !== "production") {
+    app.use(require("morgan")("combined"));
+  }
+
+  if (config.cors) {
+    app.use(require("cors")());
+  }
+
+  if (config.bodyParser) {
+    app.use(express[config.bodyParser]());
+  }
+
+  return app;
+}
+
+function startApp(
+  serviceName = "APP",
+  port = parseInt(process.env.PORT as string),
```

Comment:
@davidmeirlevy should I add ENV validation in order to get rid of the casting here?